web/styles/zulip.css

```diff
@@ -1559,20 +1561,11 @@ td.pointer {
     }
 }
 
-.recipient_row .message_row:first-child .unread_marker {
-    top: 0;
-}
-
 .unread_marker {
-    display: block;
-    position: absolute;
-    height: 100%;
-    top: 0;
+    grid-column: 1 / 2;
```

Comment:
Just use the named grid areas (`grid-area: unread_marker`). With grid areas, there's no need to put `grid-column` or `grid-row` information on grid items (and there's a potential that Very Bad Things might happen). Keep the column and row definitions up on the parent item: the grid container.